go/teams/service_helper.go

```diff
@@ -1616,6 +1616,31 @@ func ChangeTeamSettings(ctx context.Context, g *libkb.GlobalContext, teamName st
 	})
 }
 
+func ChangeTeamSettingsByID(ctx context.Context, g *libkb.GlobalContext, id keybase1.TeamID,
+	settings keybase1.TeamSettings) error {
```

Comment:
How about making `ChangeTeamSettings` resolve the ID and then pass it to `ChangeTeamSettingsByID`? Also callers of `ChangeTeamSettings` are good candidates for this same ID treatment.

Reply:
Changed about half of them, don't really feel like doing the rest, does it feel important to you

Reply:
Huh maybe I'm looking differently but now `TeamSetSettings` is the only remaining one. Not more important than other things in this PR